franka_control/src/franka_control_node.cpp

```diff
@@ -124,6 +124,11 @@ int main(int argc, char** argv) {
           control_manager.update(now, now - last_time);
           franka_control.checkJointLimits();
           last_time = now;
+
+          if (has_error && franka_control.robotMode() == franka::RobotMode::kIdle) {
+            franka_control.robot().automaticErrorRecovery();
```

Comment:
This line is unnecessary. We will remove it and merge this PR.

Please let us know if we are missing something.

Reply:
Which line exactly do you refer to? As you already force-pushed, I cannot easily see my suggested code anymore.
But in a different place, you call `automaticErrorRecovery()` as well:
https://github.com/frankaemika/franka_ros/blob/c392c9112e1184f6dad5437f4453cb8020de4219/franka_control/src/franka_control_node.cpp#L53-L54
Why is it unnecessary here?